exporters/variables-scss/src/content/generators/cssGenerator.ts

```diff
@@ -0,0 +1,94 @@
+import { DimensionToken, StringToken, Token, TokenGroup, TokenType } from '@supernovaio/sdk-exporters';
+import { CSSHelper } from '@supernovaio/export-helpers';
+import { toPlural } from '../helpers/stringHelper';
+import { tokenVariableName } from '../helpers/tokenHelper';
+import { handleExceptions } from '../helpers/exceptionHelper';
+
+export const tokenToCSSByType = (
+  token: Token,
+  mappedTokens: Map<string, Token>,
+  tokenGroups: Array<TokenGroup>,
+  withParent: boolean,
+): string | null => {
+  if (token.tokenType === TokenType.dimension) {
+    const dimensionToken = token as DimensionToken;
+    const name = tokenVariableName(dimensionToken, tokenGroups, withParent);
+    let value = dimensionToken.value?.measure;
+    value = handleExceptions(name, value);
+    const unit = CSSHelper.unitToCSS(dimensionToken.value?.unit);
+
+    return `$${name}: ${value}${unit} !default;`;
```

Comment:
maybe this can be a function too?

So you do something like formatSCSSVariables(name, `${value}${unit}`). And it returns `$${name}: ${value} !default;`